build.sh

```diff
@@ -170,7 +170,7 @@ fi
 # Determining the value for DOCKER_FROM
 ###
 if [ -z "$DOCKER_FROM" ]; then
-  DOCKER_FROM="alpine:3.14"
+  DOCKER_FROM="debian:bullseye-slim"
```

Comment:
IMO using the version number (here 11-slim) is preferred as not all are that familiar with the release names in Debian. But at least a number is monotonic.

Reply:
Ok, i will change it to the number.